lua/lvim/config/init.lua

```diff
@@ -98,6 +98,8 @@ end
 --- Override the configuration with a user provided one
 -- @param config_path The path to the configuration overrides
 function M:reload()
+  require("lvim.keymappings").clear(lvim.keys)
```

Comment:
I think you missed a critical use-case where the keymaps are overridden by a plugin and not the user. What's the plan then? Unless you reload _all_ the plugins, then I'm not sure if it's even possible.

Reply:
It's more like I don't want to define the plugin's mapping manually _if possible_. My use case is specifically about `<C-hjkl>` as they're usually used by tmux-related plugins.

Reply:
Okay, so your specific use case is that you are using a plugin that overrides a default which you aren't defining in your `lvim.keys`

So basically the fix here would be to _not_ unset any default key mapping

Reply:
Still the same problem

https://user-images.githubusercontent.com/59826753/141161515-2910d313-ba04-4c7f-90f5-507279b1293b.mp4